frontend/src/components/Admin/DevPortfolio/DevPortfolioDetails.tsx

```diff
@@ -54,7 +54,10 @@ const DevPortfolioDetails: React.FC<Props> = ({ uuid, isAdminView }) => {
         netid: submission.member.netid,
         opened_score: open,
         reviewed_score: review,
-        total_score: open + review
+        total_score:
+          (submission.status === 'valid' && 2) || // in case admin gave full points
+          (submission.status === 'invalid' && '0') || // in case admin gave zero points
+          open + review
```

Comment:
> This logic changes how submissions are deemed valid and invalid. Are we going to modify the exported CSV data to include if an admin manually changes a submission's status?

I knew I forgot something :)   
Let me know what you think of this solution: A `valid` overall status gives full points. Either they got this by having one valid PR in each category (in which case `open + review` would sum to 2), or an admin overrode the results to give them full points. An `invalid` overall status gives zero points, either by an admin override or by not having at least one valid PR in each category. In the case of partial points (im not sure how common this case is), the csv still has the individual score of each category, and so the admins could manually compute this. Or, the admins could override the status by setting it to `pending` (which admittedly would be somewhat confusing to an end user), which would default to the sum of the `open` and `review` points.

Reply:
@airaagrawal1

Reply:
Ya the `pending` status is a bit confusing, but I think this logic is good for the purposes of exporting to csv. Do you think we should include a `status` column in the csv for admins to double check `total_score` and `status` in the future? For example, an admin thinks someone's score is 2 but they marked it as pending in Idol and forgot to change it.

Reply:
Yea in the future the leads may want some way to filter the table view to see if there are any `pending` submissions, but i'll add `status` as a column in the meantime. I'll have to get feedback from them on whether that's something theyre interested in

Reply:
Sounds good